core/src/main/java/org/apache/calcite/sql/SqlWriterConfig.java

```diff
@@ -433,4 +424,6 @@ SqlWriterConfig withWhereListItemsOnSeparateLines(
     /** Wrap always. Items are on separate lines. */
     TALL
   }
+
+  static SqlWriterConfig DEFAULT = ImmutableSqlWriterConfig.DEFAULT;
```

Comment:
Do we need this?  Couldn't we simply reference ImmutableSqlWriterConfig.DEFAULT directly in SqlPrettyWriter.config?

Reply:
Modified. This isn't included anymore